homeassistant/components/wirelesstag/diagnostics.py

```diff
@@ -0,0 +1,23 @@
+"""Diagnostics support for Wireless Sensor Tag."""
```

Comment:
Please don't add any new features that aren't required for the refactor. This PR is big enough and will probably have a hard time finding reviewers with enough time to review it.

Try to break out all changes that aren't strictly needed now to separate PRs.